eng/versioning/version_client.txt

```diff
@@ -45,7 +45,7 @@ com.azure:azure-communication-common;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-administration;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-sms;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-core;1.11.0;1.12.0-beta.1
-com.azure:azure-core-amqp;1.6.0;1.7.0-beta.3
+com.azure:azure-core-amqp;2.0.0;2.1.0-beta.1
```

Comment:
Same question as above. This should be in a core-amqp post release PR?

Reply:
yes